ui-components/src/helpers/tableSummaries.tsx

```diff
@@ -1,71 +1,91 @@
 import * as React from "react"
 import { t } from "./translator"
-import { UnitSummary } from "@bloom-housing/backend-core/types"
+import { UnitSummary, UnitsSummary } from "@bloom-housing/backend-core/types"
 import { GroupedTableGroup } from "../tables/GroupedTable"
 
-export const unitSummariesTable = (summaries: UnitSummary[]) => {
-  const unitSummaries = summaries.map((unitSummary) => {
-    const minIncome =
-      unitSummary.minIncomeRange.min == unitSummary.minIncomeRange.max ? (
-        <strong>{unitSummary.minIncomeRange.min}</strong>
-      ) : (
-        <>
-          <strong>{unitSummary.minIncomeRange.min}</strong> {t("t.to")}{" "}
-          <strong>{unitSummary.minIncomeRange.max}</strong>
-        </>
-      )
+export const getSummaryRow = (
+  minIncomeRangeMin: string | undefined,
```

Comment:
Another way to do this (and maybe a more standard way to do this?) is to add the `?`, so this would be

```
minIncomeRangeMin?: string
```

Source: https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-0.html#optional-parameters-and-properties